rewriter/cleanup.cc

```diff
@@ -0,0 +1,45 @@
+#include "rewriter/cleanup.h"
+#include "ast/Helpers.h"
+#include "ast/ast.h"
+#include "ast/treemap/treemap.h"
+#include "core/core.h"
+
+using namespace std;
+
+namespace sorbet::rewriter {
+
+// This pass gets rid of some unnecessary nodes that are likely to have gotten created in the course of the rewriter
+// pass, specifically by removing EmptyTree nodes whenever they are created
```

Comment:
This makes it sound like all `EmptyTree` nodes get removed, is it just specific cases where they're removed?

Reply:
Updated this comment.